problem/src/main/java/org/zalando/problem/Status.java

```diff
@@ -268,6 +270,19 @@
         this.reason = reasonPhrase;
     }
 
+    /**
+     * Creates a Status instance from the given code.
+     *
+     * @param code the HTTP code as a number
+     * @return the correct enum value for this status code.
```

Comment:
This should mention a `@throws` section, imo.

Reply:
Good catch, I'll add it.